configure.py

```diff
@@ -0,0 +1,28 @@
+''' writes a makefile after finding LDIC '''
+import platform as p;import sys as s;import sysconfig as c;import subprocess as sb;import os;op=os.path;S=op.sep;js=lambda x:" ".join(x);sp=lambda x:x.split()
+def ON(x):print(x);return x
+def E(x):s.stderr.write("error: "+x+"\n");exit(1)
+V,v=Vv=s.version_info[:2];a=p.machine();o=s.platform;sv=s.version;pv=p.version()
+if V<3:E('version')
+if "win32" in o:E('nyi')
+try:import numpy as n
+except:import traceback as tb;tb.print_exc();E('numpy')
+
+gcc=lambda x:"-fPIC -shared" if x[0]=="l" else "-bundle -undefined dynamic_lookup"
+pvm='python%d.%dm'%Vv;dy=lambda x:'lib'+x+'.so'
```

Comment:
"m" in "3.6m" is and ABI flag that indicates python configured using a `--with-pymalloc` option.  While this is the default for most distributions, I would not hardcode that.  Consider using `distutils.sysconfig.get_config_var('ABIFLAGS')` instead.  There is also `distutils.sysconfig.get_config_var('LDVERSION')` which seems to expand to `3.6m` directly.